sections/image-with-text.liquid

```diff
@@ -1,6 +1,5 @@
 {{ 'component-image-with-text.css' | asset_url | stylesheet_tag }}
-
-<div class="image-with-text {% if section.settings.full_width %}image-with-text--full-width{% else %}page-width{% endif %} color-scheme-{{ section.settings.color_scheme }}">
+<div class="image-with-text {% if section.settings.full_width %}image-with-text--full-width{% else %}page-width{% endif %} color-scheme-{{ section.settings.color_scheme }}{% if settings.text_boxes_border_thickness > 0 and settings.text_boxes_border_opacity > 0 and settings.media_border_thickness > 0 and settings.media_border_opacity > 0 %} collapse-borders{% endif %}{% unless section.settings.color_scheme == 'background-1' and settings.media_border_thickness > 0 and settings.text_boxes_shadow_opacity == 0 and settings.text_boxes_border_thickness == 0 or settings.text_boxes_border_opacity == 0 %} collapse-corners{% endunless %}">
```

Comment:
Hmm yeah I wonder how long that's been in there. Must've had a full width setting at some point. If I have to make other edits I can remove, otherwise I'll catch it when we have to merge into Martina's image with text overlay updates.